ai/src/main/kotlin/ktx/ai/behaviorTree.kt

```diff
@@ -0,0 +1,441 @@
+package ktx.ai
+
+import com.badlogic.gdx.ai.btree.BehaviorTree
+import com.badlogic.gdx.ai.btree.Task
+import com.badlogic.gdx.ai.btree.branch.DynamicGuardSelector
+import com.badlogic.gdx.ai.btree.branch.Parallel
+import com.badlogic.gdx.ai.btree.branch.Parallel.Orchestrator
+import com.badlogic.gdx.ai.btree.branch.Parallel.Policy
+import com.badlogic.gdx.ai.btree.branch.RandomSelector
+import com.badlogic.gdx.ai.btree.branch.RandomSequence
+import com.badlogic.gdx.ai.btree.decorator.AlwaysFail
+import com.badlogic.gdx.ai.btree.decorator.AlwaysSucceed
+import com.badlogic.gdx.ai.btree.decorator.Include
+import com.badlogic.gdx.ai.btree.decorator.Invert
+import com.badlogic.gdx.ai.btree.decorator.Repeat
+import com.badlogic.gdx.ai.btree.decorator.SemaphoreGuard
+import com.badlogic.gdx.ai.btree.decorator.UntilFail
+import com.badlogic.gdx.ai.btree.decorator.UntilSuccess
+import com.badlogic.gdx.ai.btree.leaf.Failure
+import com.badlogic.gdx.ai.btree.leaf.Success
+import com.badlogic.gdx.ai.btree.leaf.Wait
+import com.badlogic.gdx.ai.utils.random.ConstantFloatDistribution
+import com.badlogic.gdx.ai.utils.random.ConstantIntegerDistribution
+import com.badlogic.gdx.ai.utils.random.FloatDistribution
+import com.badlogic.gdx.ai.utils.random.IntegerDistribution
+import com.badlogic.gdx.utils.Array
+import kotlin.contracts.ExperimentalContracts
+import kotlin.contracts.InvocationKind
+import kotlin.contracts.contract
+
+/** Alias for [com.badlogic.gdx.ai.btree.branch.Sequence] avoiding name collision with the standard library. */
+typealias GdxAiSequence<E> = com.badlogic.gdx.ai.btree.branch.Sequence<E>
+
+/** Alias for [com.badlogic.gdx.ai.btree.branch.Selector] avoiding name collision with the standard library. */
+typealias GdxAiSelector<E> = com.badlogic.gdx.ai.btree.branch.Selector<E>
+
+/** Alias for [com.badlogic.gdx.ai.btree.decorator.Random] avoiding name collision with the standard library. */
+typealias GdxAiRandom<E> = com.badlogic.gdx.ai.btree.decorator.Random<E>
+
+/** Should annotate builder methods of gdxAI [Task]. */
+@DslMarker
+@Target(
+  AnnotationTarget.CLASS,
+  AnnotationTarget.TYPE_PARAMETER,
+  AnnotationTarget.FUNCTION,
+  AnnotationTarget.TYPE,
+  AnnotationTarget.TYPEALIAS
+)
+annotation class GdxAiTaskDsl
```

Comment:
If you don't plan on introducing multiple DSLs that could interfere with each other, I think `GdxAiDsl` should be fine.